flags.go

```diff
@@ -169,22 +165,19 @@ func newApp() (app *cli.App) {
 			cli.Float64Flag{
 				Name:  "limit-bytes-per-sec",
 				Value: -1,
-				Usage: "Bandwidth limit for reading data, measured over a 30-second " +
-					"window. (use -1 for no limit)",
+				Usage: "Bandwidth limit for reading data, measured over a 30-second window. (use -1 for no limit)",
```

Comment:
add default here (as it is there in the doc).  Also, please check for other helps as well.

Reply:
default value is printing without putting it explicitly in usage

Reply:
In that case, either we should have for all flags or for None.

Reply:
Yeah (use -1 for no limit) mentioned in user guide apart from default value.